### ISSUE TITLE
Cannot start service unless I reboot.

### ISSUE THREAD

**Issue by devilswarrior** [MathewSachin/Fate-Grand-Automata#213]
Hey there,

App works great just it seems that unless I reboot my phone I cannot start the actual service it will tell me accessibility access not granted go to settings and then turn it off then on again if it's on.

I go to settings and let's say it's on, then i turn it off it will say this will stop the app I say yes then turn it back on hit allow at the prompt and then it won't work. Then I'll reboot my phone and it works fine. I can then leave the service on turn it off and on for a quick session of spending my AP and then when I try a few hours later same exact issue.

I'm running the NA version of FGO on a xiaomi mi mix 2s with their Android skin MIUI at version 11.0.5 and Android version 10.

Let me know if there's anything else you need me to do to give you more information.

**Comment by MathewSachin** [comment]
I use a Xiaomi phone myself and don't have any problems after disabling battery optimizations and all for the app.
Just do everything this site says: https://dontkillmyapp.com/xiaomi

**Comment by devilswarrior** [comment]
That looks like it worked I wasn't able to do everything on that page
probably due to a different MIUI version but after doing the rest I haven't
had any issues, thanks.
